examples/contrib/cifar10/main.py

```diff
@@ -187,6 +187,12 @@ def run(
         **spawn_kwargs: Other kwargs to spawn run in child processes: master_addr, master_port, node_rank, nnodes
 
     """
+    # check to see if the num_epochs is greater than or equal to num_warmup_epochs
+    if num_warmup_epochs >= num_epochs:
```

Comment:
If `black` doesn't work as expected, or whatever, please try a correction like 
```python
    if num_warmup_epochs >= num_epochs:
        raise ValueError(
            "num_epochs cannot be less than or equal to num_warmup_epochs, please increase num_epochs "
            "or decrease num_warmup_epochs"
        )
```
and let's see.